src/uct/ib/ud/base/ud_iface.h

```diff
@@ -35,6 +35,8 @@ BEGIN_C_DECLS
 
 enum {
     UCT_UD_IFACE_STAT_RX_DROP,
+    UCT_UD_IFACE_STAT_RX_COMPLETION,
+    UCT_UD_IFACE_STAT_TX_COMPLETION,
```

Comment:
to common code

Reply:
@ivankochin also - better to reflect this in the PR title and/or description (it adds not *only* zipping stats).